From: hololeap <hololeap@protonmail.com>
Signed-off-by: hololeap <hololeap@protonmail.com>

These patches allow the tests to compile. An older version of tasty-hspec may
have exported Test.Hspec automatically, but the current version doesn't seem to.

diff -urN purescript-0.14.1-orig/tests/TestBundle.hs purescript-0.14.1-tests/tests/TestBundle.hs
--- purescript-0.14.1-orig/tests/TestBundle.hs	2021-04-15 07:51:28.000000000 -0600
+++ purescript-0.14.1-tests/tests/TestBundle.hs	2021-05-13 18:07:07.459372176 -0600
@@ -31,6 +31,7 @@
 import TestUtils
 import Test.Tasty
 import Test.Tasty.Hspec
+import Test.Hspec
 
 main :: IO TestTree
 main = testSpec "bundle" spec
diff -urN purescript-0.14.1-orig/tests/TestCompiler.hs purescript-0.14.1-tests/tests/TestCompiler.hs
--- purescript-0.14.1-orig/tests/TestCompiler.hs	2021-04-15 07:51:28.000000000 -0600
+++ purescript-0.14.1-tests/tests/TestCompiler.hs	2021-05-13 18:07:59.719934472 -0600
@@ -56,6 +56,7 @@
 import Test.Tasty
 import Test.Tasty.Hspec
 import Test.Tasty.Golden (goldenVsString)
+import Test.Hspec
 
 main :: IO TestTree
 main = do
diff -urN purescript-0.14.1-orig/tests/TestCoreFn.hs purescript-0.14.1-tests/tests/TestCoreFn.hs
--- purescript-0.14.1-orig/tests/TestCoreFn.hs	2021-04-15 07:51:28.000000000 -0600
+++ purescript-0.14.1-tests/tests/TestCoreFn.hs	2021-05-13 18:14:27.344120687 -0600
@@ -24,6 +24,7 @@
 
 import Test.Tasty
 import Test.Tasty.Hspec
+import Test.Hspec
 
 main :: IO TestTree
 main = testSpec "corefn" spec
diff -urN purescript-0.14.1-orig/tests/TestDocs.hs purescript-0.14.1-tests/tests/TestDocs.hs
--- purescript-0.14.1-orig/tests/TestDocs.hs	2021-04-15 07:51:28.000000000 -0600
+++ purescript-0.14.1-tests/tests/TestDocs.hs	2021-05-13 18:10:57.301849379 -0600
@@ -30,7 +30,8 @@
 import TestPscPublish (preparePackage)
 
 import Test.Tasty
-import Test.Tasty.Hspec (Spec, it, context, expectationFailure, runIO, testSpec)
+import Test.Tasty.Hspec
+import Test.Hspec hiding (before)
 
 main :: IO TestTree
 main = testSpec "docs" spec
diff -urN purescript-0.14.1-orig/tests/TestGraph.hs purescript-0.14.1-tests/tests/TestGraph.hs
--- purescript-0.14.1-orig/tests/TestGraph.hs	2021-04-15 07:51:28.000000000 -0600
+++ purescript-0.14.1-tests/tests/TestGraph.hs	2021-05-13 18:13:55.883780142 -0600
@@ -5,6 +5,7 @@
 
 import Test.Tasty
 import Test.Tasty.Hspec
+import Test.Hspec
 import System.IO.UTF8 (readUTF8FileT)
 import Data.Either (isLeft)
 
diff -urN purescript-0.14.1-orig/tests/TestHierarchy.hs purescript-0.14.1-tests/tests/TestHierarchy.hs
--- purescript-0.14.1-orig/tests/TestHierarchy.hs	2021-04-15 07:51:28.000000000 -0600
+++ purescript-0.14.1-tests/tests/TestHierarchy.hs	2021-05-13 18:13:21.083403571 -0600
@@ -7,7 +7,8 @@
 import qualified Language.PureScript as P
 
 import Test.Tasty
-import Test.Tasty.Hspec (describe, it, shouldBe, testSpec)
+import Test.Tasty.Hspec
+import Test.Hspec
 
 main :: IO TestTree
 main = testSpec "hierarchy" $ do
diff -urN purescript-0.14.1-orig/tests/TestIde.hs purescript-0.14.1-tests/tests/TestIde.hs
--- purescript-0.14.1-orig/tests/TestIde.hs	2021-04-15 07:51:28.000000000 -0600
+++ purescript-0.14.1-tests/tests/TestIde.hs	2021-05-13 18:12:37.662933927 -0600
@@ -7,6 +7,7 @@
 import qualified PscIdeSpec
 import           Test.Tasty
 import           Test.Tasty.Hspec
+import           Test.Hspec
 
 main :: IO TestTree
 main =
diff -urN purescript-0.14.1-orig/tests/TestMake.hs purescript-0.14.1-tests/tests/TestMake.hs
--- purescript-0.14.1-orig/tests/TestMake.hs	2021-04-15 07:51:28.000000000 -0600
+++ purescript-0.14.1-tests/tests/TestMake.hs	2021-05-13 18:11:45.022364889 -0600
@@ -28,6 +28,7 @@
 
 import Test.Tasty
 import Test.Tasty.Hspec
+import Test.Hspec
 
 utcMidnightOnDate :: Integer -> Int -> Int -> UTCTime
 utcMidnightOnDate year month day = UTCTime (fromGregorian year month day) (secondsToDiffTime 0)
diff -urN purescript-0.14.1-orig/tests/TestPrimDocs.hs purescript-0.14.1-tests/tests/TestPrimDocs.hs
--- purescript-0.14.1-orig/tests/TestPrimDocs.hs	2021-04-15 07:51:28.000000000 -0600
+++ purescript-0.14.1-tests/tests/TestPrimDocs.hs	2021-05-13 18:01:56.926047736 -0600
@@ -11,8 +11,8 @@
 import qualified Language.PureScript.Docs as D
 
 import Test.Tasty
-import Test.Tasty.Hspec (Spec, testSpec, it)
-import Test.Hspec (shouldBe)
+import Test.Tasty.Hspec
+import Test.Hspec
 
 main :: IO TestTree
 main = testSpec "prim docs" spec
diff -urN purescript-0.14.1-orig/tests/TestPscPublish.hs purescript-0.14.1-tests/tests/TestPscPublish.hs
--- purescript-0.14.1-orig/tests/TestPscPublish.hs	2021-04-15 07:51:28.000000000 -0600
+++ purescript-0.14.1-tests/tests/TestPscPublish.hs	2021-05-13 18:09:39.511009813 -0600
@@ -25,8 +25,9 @@
 import qualified Language.PureScript.Publish.ErrorsWarnings as Publish
 
 import Test.Tasty
-import Test.Tasty.Hspec (Spec, Expectation, runIO, context, it, expectationFailure, testSpec)
+import Test.Tasty.Hspec
 import TestUtils hiding (inferForeignModules, makeActions)
+import Test.Hspec hiding (before)
 
 main :: IO TestTree
 main = testSpec "publish" spec
diff -urN purescript-0.14.1-orig/tests/TestUtils.hs purescript-0.14.1-tests/tests/TestUtils.hs
--- purescript-0.14.1-orig/tests/TestUtils.hs	2021-04-15 07:51:28.000000000 -0600
+++ purescript-0.14.1-tests/tests/TestUtils.hs	2021-05-13 18:06:55.429242831 -0600
@@ -32,7 +32,7 @@
 import System.FilePath
 import qualified System.FilePath.Glob as Glob
 import System.IO
-import Test.Tasty.Hspec
+import Test.Hspec
 
 
 findNodeProcess :: IO (Maybe String)
